scripts/update_readme_for_theme.py

```diff
@@ -16,44 +21,66 @@
-
 HERE = Path(__file__).parent
 REPO = HERE.parent
+sys.path.append(str(REPO / "test"))
+from render_html import outdir as html_outdir  # noqa: E402
+from render_html import render_html  # noqa: E402
 
 env = Environment(
     loader=FileSystemLoader(HERE / "templates"), autoescape=select_autoescape()
 )
 
 
-def update_readme(theme):
-    """Given a theme module name, update that theme's README file on disk"""
-
-    # Get the theme colors
-    theme_style_module = import_module(f"a11y_pygments.{theme}.style")
-    color_cls = theme_style_module.Colors  # access to foreground colors
-    theme_cls = theme_style_module.Theme  # access to docstring
-    theme_kebab_case = theme.replace("_", "-")
-    style = get_style_by_name(
-        theme_kebab_case
-    )  # access to background and highlight colors
+def contrast_by_foreground_color(color_cls, background_color: str):
+    """Calculate contrast ratios and WCAG ratings for all foreground colors
 
-    # Calculate contrast ratios and WCAG ratings for all foreground colors
+    :param color_cls: A mapping of color names to six-value hex CSS color
+        strings, the mapping comes from a class named Colors defined in
+        <theme-name>/style.py
+    :type color_cls: class object
+    :param str background_color: Theme default background color for code blocks
+        in six-value #RRGGBB hex format; for example white would be "#ffffff"
+        (case insensitive)
+    """
     foreground_colors = {}
     for key in vars(color_cls):
         value = getattr(color_cls, key)
         if not callable(value) and not key.startswith("__"):
             if value in foreground_colors:
                 continue
-            contrast = contrast_ratio(
-                hex_to_rgb(value), hex_to_rgb(style.background_color)
-            )
+            contrast = contrast_ratio(hex_to_rgb(value), hex_to_rgb(background_color))
             foreground_colors[value] = {
                 "hex": hexstr_without_hash(value),
                 "contrast_ratio": round(contrast, 1),
                 "wcag_level_normal_text": get_wcag_level_normal_text(contrast),
                 "wcag_level_large_text": get_wcag_level_large_text(contrast),
             }
+    return foreground_colors
+
+
+def update_readme(theme):
+    """Given a theme module name, update that theme's README file on disk"""
+
+    theme_kebab_case = theme.replace("_", "-")
+    outdir = REPO / "a11y_pygments" / theme
+
+    # Take a screenshot of the theme applied to a sample bash script
+    with sync_playwright() as p:
+        browser = p.chromium.launch()
+        page = browser.new_page(viewport={"width": 620, "height": 720})
+        bash_sample_rendered_html = html_outdir / theme_kebab_case / "bash.html"
+        page.goto("file://" + bash_sample_rendered_html.absolute().as_posix())
+        page.screenshot(path=outdir / "images" / f"{theme_kebab_case}.png")
+        browser.close()
```

Comment:
I looked into [shot-scraper](https://github.com/simonw/shot-scraper) but it seemed better to me to just add Playwright directly as a dependency and write these 7 lines of code.